src/current/_includes/cockroachcloud/backups/terraform-managed-backups.md

```diff
@@ -0,0 +1,9 @@
+Set the following for the `backup_config` attribute:
+
+- `enabled` controls whether managed backups are enabled or disabled. If you are disabling managed backups, you cannot set backup frequency or retention. Possible values are: `true` or `false`.
+- `frequency_minutes` determines [how often](#frequency) the managed backup will run in minutes. Possible values are: `5`, `10`, `15`, `30`, `60`, `240`, `1440`.
```

Comment:
we don't do this anywhere else but I wonder if it's helpful to add the hours in parens for the last 2?   i.e.
```
Possible values are: `5`, `10`, `15`, `30`, `60`, `240` (4 hours), `1440` (24 hours).
```

Reply:
I like this a lot! I added to this section in the Terraform section here, and to the same in the Cloud API section.